tests/recommendations_endpoint_test.py

```diff
@@ -136,6 +136,58 @@ def test_user_selects_price_preference_4_returns_successful_response():
     assert price == 4
     assert response_body['data']['attributes']['is_closed'] == False
 
+def test_user_enters_only_full_address_and_gets_successful_response():
+    address = '1331 17th St, Denver, CO'
+    url = '/api/v1/recommendations?address={}'.format(address)
+    client = app.test_client()
+    response = client.get(url)
+    assert response.status_code == 200
+    response_body = response.json
+    assert 'id' in response_body['data']
+    assert 'cuisine' in response_body['data']['attributes']
+    assert 'name' in response_body['data']['attributes']
+    assert 'location' in response_body['data']['attributes']
+    assert 'phone' in response_body['data']['attributes']
+    assert 'display_phone' in response_body['data']['attributes']
+    assert 'rating' in response_body['data']['attributes']
+    assert 'price' in response_body['data']['attributes']
+    assert response_body['data']['attributes']['is_closed'] == False
+
+def test_user_enters_city_and_gets_successful_response():
+    address = 'Denver'
+    url = '/api/v1/recommendations?address={}'.format(address)
+    client = app.test_client()
+    response = client.get(url)
+    assert response.status_code == 200
+    response_body = response.json
+    assert 'id' in response_body['data']
+    assert 'cuisine' in response_body['data']['attributes']
+    assert 'name' in response_body['data']['attributes']
+    assert 'location' in response_body['data']['attributes']
+    assert 'phone' in response_body['data']['attributes']
+    assert 'display_phone' in response_body['data']['attributes']
+    assert 'rating' in response_body['data']['attributes']
+    assert 'price' in response_body['data']['attributes']
+    assert response_body['data']['attributes']['is_closed'] == False
+
+def test_user_enters_state_and_gets_successful_response():
+    address = 'Colorado'
+    url = '/api/v1/recommendations?address={}'.format(address)
+    client = app.test_client()
+    response = client.get(url)
+    assert response.status_code == 200
+    response_body = response.json
+    assert 'id' in response_body['data']
+    assert 'cuisine' in response_body['data']['attributes']
+    assert 'name' in response_body['data']['attributes']
+    assert 'location' in response_body['data']['attributes']
+    assert 'phone' in response_body['data']['attributes']
+    assert 'display_phone' in response_body['data']['attributes']
+    assert 'rating' in response_body['data']['attributes']
+    assert 'price' in response_body['data']['attributes']
+    assert response_body['data']['attributes']['is_closed'] == False
+
+
```

Comment:
Should there also be a test for full address, including zip code?

Reply:
We can definitely add a test in for this!